apps/api/v2/test/setEnvVars.ts

```diff
@@ -1,4 +1,5 @@
 import type { Environment } from "@/env";
+import "dotenv/config";
```

Comment:
Added this because without it the e2e tests weren't considering the env variables.